readme.md

```diff
@@ -41,17 +41,20 @@ Add the following to your java invocation. (Tomcat users can use the `setenv.sh`
 -javaagent:/abspath/to/luceedebug.jar=jdwpHost=localhost,jdwpPort=9999,debugHost=0.0.0.0,debugPort=10000,jarPath=/abspath/to/luceedebug.jar
 ```
 
-Most java devs will be familiar with the `agentlib` part. We need JDWP running, listening on a socket on localhost. luceedebug will attach to JDWP over a socket, running from within the same JVM. It stays attached for the life of the JVM.
-
-Note that JDWP `address` and luceedebug's `jdwpHost`/`jdwpPort` must match.
-
-The `debugPort` and `debugHost` options are the host/port that the VS Code debugger attaches to. Note that JDWP can usually listen on localhost (the connection to JDWP from luceedebug is a loopback connection).
-
-If Lucee is running in a docker container, the `debugHost` must be `0.0.0.0`. However, be careful not to do this on a publicly-accessible, unprotected server, as you could expose the debugger to the public (which would be a major security vulnerability).
-
-The `jarPath` argument is the absolute path to the luceedebug.jar file. Unfortunately we have to say its name twice! One tells the JVM which jar to use as a java agent, the second is an argument to the java agent about where to find the jar it will load debugging instrumentation from.
-
-(There didn't seem to be an immediately obvious way to pull the name of "the current" jar file from an agent's `premain`, but maybe it's just been overlooked. If you know let us know!)
+* `agentlib`: Configures JDWP, which is the lower level Jave debugging protocol that the luceedebug agent connects with. (Note: The VS Code debugger connects to the _luceedebug_ agent, not JDWP, so JDWP/`agentlib` usually doesn't need to be modified/customized.)

```

Comment:
nit: `Jave` -> `Java`

Reply:
Done. Also did a bit of a punctuation/grammar scrub.